src/modules/feature-modules/accessor/pages/innovation/overview/overview.component.ts

```diff
@@ -79,6 +79,17 @@ export class InnovationOverviewComponent extends CoreComponent implements OnInit
 
       this.innovationSummary = [
         { label: 'Company', value: innovationInfo.owner?.organisation?.name ?? 'No company' },
+        ...(this.innovation.owner?.organisation?.size
+          ? [{ label: 'Company size', value: this.innovation.owner?.organisation?.size ?? '' }]
```

Comment:
I don't think you need the default value and neither the verifications since you are verifying it before.

Hopefully, `value: this.innovation.owner.organisation.size` should be enough!